flow-defs/image.js.flow

```diff
@@ -1,8 +1,10 @@
 // @flow
 
 import * as React from "react";
+declare export var useDisableBorderRadius: () => boolean;
+declare export var DisableBorderRadiusProvider: React.ComponentType<{}>;
 declare type AspectRatio = "1:1" | "16:9" | "7:10";
-declare type Props = {
+export type ImageProps = {
  url: string,
```

Comment:
would be nice to have the same way to call things between image and video? For example, in image componente we call "url" but in video we call it "src"

![image](https://user-images.githubusercontent.com/6722153/149497472-4156b5cb-14c9-4bfb-a568-5afc06b57db3.png)

On the other hand, make sense to add aspect ratio props like image in video? To keep both component similar

Reply:
I prefer src, because you're defining a "source". Changing the image prop would be a breaking change.

For videos the aspect ratio is not needed (at least for now) because video dimensions are known

Reply:
+1 to src and deprecate url